sycl/include/sycl/handler.hpp

```diff
@@ -2646,6 +2646,14 @@ class __SYCL_EXPORT handler {
     else
       commonUSMFill2DFallbackKernel(Dest, DestPitch, Pattern, Width, Height);
   }
+  /// Read from or write to host pipes given a host address and
+  /// \param Name name of the host pipe to be passed into lower level runtime
+  /// \param Ptr host pointer of host pipe as identified by address of its const
+  /// expr __pipe member \param Size the size of data getting read back / to.
+  /// /// \param Size the size of data getting read back / to. \param Blocking
+  /// if read/write opeartion is blocking \param Read 1 for read, 0 for write
+  void read_write_host_pipe(const std::string &Name, void *Ptr, size_t Size,
+                            bool Block, bool Read);
```

Comment:
I don't see this function described in the [sycl_ext_intel_dataflow_pipes extension](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_intel_dataflow_pipes.asciidoc). Is there a separate PR adding it?

Also, I believe adding new members to existing SYCL interfaces mandates that you prefix them with `ext_intel_` (for `ext::intel` extensions). `device_global` was an exception as it added overloads to `memcpy`/`copy`.

Reply:
Thanks Steffen. I am changing the name accordingly. Yes, there will be a PR making the change to the Spec. There was one here: https://github.com/intel/llvm/pull/5838/files, but it's outdated. I need to update it and create a new PR for it.

Reply:
I am keeping the name as ext_intel_read_write_host_pipe instead of ext_intel_readWriteHostPipe for consistency.

Reply:
It would be best to have that up ASAP. Normally we would need that reviewed first before adding new interfaces.

> I am keeping the name as ext_intel_read_write_host_pipe instead of ext_intel_ReadWriteHostPipe for consistency.

Yes, `ext_intel_read_write_host_pipe` would be more in line with other SYCL interfaces.

Reply:
Thanks @steffenlarsen. Yes, I am preparing the Spec review PR ASAP.

Reply:
@steffenlarsen , I have just created a draft [PR ](https://github.com/intel/llvm/pull/8789)for the Spec update which includes the `ext_intel_read_write_host_pipe` function. I am still working on some wording on the Spec change.